politeiawww/www.go

```diff
@@ -608,6 +608,50 @@ func (p *politeiawww) handleResetPassword(w http.ResponseWriter, r *http.Request
 	util.RespondWithJSON(w, http.StatusOK, rpr)
 }
 
+// handleProposalAccessTime returns the proposal access time for a given user
+func (p *politeiawww) handleProposalAccessTime(w http.ResponseWriter, r *http.Request) {
+	log.Tracef("handleProposalAccessTime")
+	user, err := p.getSessionUser(w, r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleProposalAccessTime: getSessionUser %v", err)
+		return
+	}
+	reply, err := p.backend.ProcessProposalAccessTime(user.Email)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleProposalAccessTime: proposalAccessTime %v", err)
+	}
+	util.RespondWithJSON(w, http.StatusOK, reply)
+}
+
+// handleNewProposalAccessTime returns the proposal access time for a user
+func (p *politeiawww) handleNewProposalAccessTime(w http.ResponseWriter, r *http.Request) {
+	log.Tracef("handleNewProposalAccessTime")
+	var pat v1.ProposalAccessTime
+	decoder := json.NewDecoder(r.Body)
+	log.Tracef("%v", decoder)
+	if err := decoder.Decode(&pat); err != nil {
+		RespondWithError(w, r, 0, "handleProposalAccessTime: unmarshal", v1.UserError{
+			ErrorCode: v1.ErrorStatusInvalidInput,
+		})
+		return
+	}
+	user, err := p.getSessionUser(w, r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleProposalAccessTime: getSessionUser %v", err)
+		return
+	}
+	log.Tracef("%v", user.Email)
+	// err = p.backend.ProcessNewProposalAccessTime(user.Email, decoder)
```

Comment:
You need to activate this code in order to properly test if you're getting things into DB. Also, you should be passing the proposal token grabbed from the URL here and not the decoder.